sdk-springboot/src/test/java/io/dapr/springboot/DaprBeanPostProcessorIT.java

```diff
@@ -0,0 +1,87 @@
+package io.dapr.springboot;
+
+import io.dapr.Topic;
+import org.assertj.core.api.WithAssertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@ContextConfiguration(
+        classes = { DaprAutoConfiguration.class }
+)
+@ExtendWith(SpringExtension.class)
+@WebAppConfiguration
+public class DaprBeanPostProcessorIT implements WithAssertions {
```

Comment:
Please, write integration tests in the sdk-tests project.

Reply:
Done! In fact, I changed an existing test; all tests used `path = "..."`, and now one test does not.

Out of curiosity, why is that module not included in the parent POM?

Reply:
Also, if there are any instructions on how to run the integration tests locally, I'd appreciate it. Now I'm hoping for the GH Actions to do their job, I couldn't get them to run locally...

Reply:
sdk-tests is not included in the main pom because it would add a circular dependency between the integration tests and the main SDK code. The integration tests require the SDK jar to be built.

To run the integration tests locally, you basically need to have dapr installed and the sdk installed via mvn install. Then you can do mvn test in the sdk-tests folder.

Reply:
Thanks, I was able to run them. I don't fully understand the circular dependency remark yet, will investigate further.